src/types/StatusState.ts

```diff
@@ -8,14 +8,19 @@ export enum StatusKey {
 
 export type Status = { [K in StatusKey]?: string };
 
-export interface Component {
+export interface ExternalServiceInfo {
   name: string;
   version?: string;
   url?: string;
 }
 
+export interface IstioEnvironment {
+  isMaistra: boolean;
```

Comment:
It's fine but I sort of wish this were an Enum like "distribution" or something like that.

Reply:
we could add something like that. the enum values would need to be propagated to the UI as well. I can look into seeing how to do this.

Reply:
Not needed for this PR, just something to think about going forward.